content/docs/operating/configuration.md

```diff
@@ -619,6 +619,15 @@ basic_auth:
 # TLS configuration.
 tls_config:
   [ <tls_config> ]
+
+# Optional namespace discovery. Currently, only a static list of namespaces
+# is supported. If set, discovery will be limited to these namespaces. If not
+# set, then all namespaces are used. Nodes are not namespaced, so this option is
+# ignored when `role: node` is set.
+namespaces:
```

Comment:
This isn't following the format we use for these. This is a config reference, not an example.